plugins/bitrot/init.go

```diff
@@ -0,0 +1,53 @@
+package bitrot
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/servers/rest/route"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/prashanthpai/sunrpc"
+)
+
+// Plugin is a structure which implements GlusterdPlugin interface
+type Plugin struct {
+}
+
+// Name returns name of plugin
+func (p *Plugin) Name() string {
+	return "bitrot"
+}
+
+// SunRPCProgram returns sunrpc program to register with Glusterd
+func (p *Plugin) SunRPCProgram() sunrpc.Program {
+	return nil
+}
+
+// RestRoutes returns list of REST API routes to register with Glusterd
+func (p *Plugin) RestRoutes() route.Routes {
+	return route.Routes{
+		route.Route{
+			Name:        "BitrotHelp",
+			Method:      "GET",
+			Pattern:     "/bitrot/help",
+			Version:     1,
+			HandlerFunc: bitrotHelpHandler},
```

Comment:
This route can be removed, added just for example route

Reply:
Done